sphinxcontrib/spelling/filters.py

```diff
@@ -230,4 +229,4 @@ def _get_contributors(self):
             return set()
         output = p.stdout.decode('utf-8')
         tokenizer = get_tokenizer('en_US', filters=[])
-        return set(word for word, pos in tokenizer(output))
+        return {word for word, pos in tokenizer(output)}
```

Comment:
I'm not especially a fan of this literal style, because it takes careful reading to notice the difference between a set comprehension and a dict comprehension. I won't hold up the PR over it, though, and we'll see if I actually end up tripping over it when I've had a chance to get used to it. :-)